--- google-gadgets-for-linux-0.10.4/configure.ac	2008-12-16 03:51:48.270190041 +0200
+++ google-gadgets-for-linux-0.10.4/configure.ac	2008-12-16 09:54:07.710174294 +0200
@@ -917,6 +917,7 @@
   GTKMOZEMBED_CFLAGS="$GTKMOZEMBED_CFLAGS -I$EMBED_INCDIR/widget"
   GTKMOZEMBED_CFLAGS="$GTKMOZEMBED_CFLAGS -I$EMBED_INCDIR/gtkembedmoz"
   GTKMOZEMBED_CFLAGS="$GTKMOZEMBED_CFLAGS -I$EMBED_INCDIR/xpcom"
+  GTKMOZEMBED_CFLAGS="$GTKMOZEMBED_CFLAGS -I$EMBED_INCDIR/js"
   GTKMOZEMBED_LDFLAGS=""
   MOZILLA_FIVE_HOME=`$PKG_CONFIG --variable=libdir $has_gtkmozembed`
   if test "x$MOZILLA_FIVE_HOME" = "x"; then
@@ -947,6 +948,9 @@
       LIBMOZJS_CFLAGS="$LIBMOZJS_CFLAGS -I$LIBMOZJS_INCDIR/unstable"
     fi
   fi
+  if test -f $LIBMOZJS_INCDIR/js/jsapi.h; then
+    LIBMOZJS_CFLAGS="$LIBMOZJS_CFLAGS -I$LIBMOZJS_INCDIR/js"
+  fi
 
   # Find out the real path of libmozjs.so
   LIBMOZJS_LIBDIR=`$PKG_CONFIG --variable=libdir $has_libmozjs`
